code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -141,6 +141,12 @@
 	/// Current amount of cannibalise charges
 	var/cannibalise_charges = 0
 
+/datum/action/xeno_action/create_spiderling/New(Target)
```

Comment:
Why is this in `New`. What if the widow matures? Surely there's a cleaner way.

Reply:
I'll move it to give_action, good point, I couldn't find a initialize so I mistakenly used new